app/src/main/res/layout/fragment_launch_screen.xml

```diff
@@ -76,7 +76,7 @@
 
     <Button
         android:id="@+id/button"
-        style="@style/App_button"
+        style="@style/StyledButton"
```

Comment:
Are you sure those changes should be in this PR?

Reply:
The program would not compile without these changes. Someone renamed the style for Button and it didn't change the name in these layouts. In such a situation, change it so that the program compiles even though it is not my PR or leave it with an error?